### PR TITLE
Initial upload

### PR DESCRIPTION
**Pull request by jdfreder**

### CODE REVIEW

**Comment by rgbkrk** [comment]
:boom:

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  return new Promise((resolve, reject) => {
+    request(url, function (error, response, body) {
+      if (!error && response.statusCode == 200) {
+        resolve(body);
+      } else {
+        reject(body);
+      }
+    });
+  });
+}
```
Comment (rgbkrk) [inline src/index.js]: I'd use the fetch poly fill in the future for this, since you're relying on promises anyway here.

**Comment by rgbkrk** [comment]
Cool stuff, looking forward to the active connection.

File: src/index.js (hunk 1 above)
Comment (jdfreder) [inline src/index.js]: Aw!  I forgot about that API, shoot.  Feel free to PR that in...

**Pull request merged**